Q3-2023 community board update (#1998)
writethedocs/www: docs/blog/2023-Q3-community-board.rst

.. post:: Aug 3, 2023
   :tags: community board, news, report
   :author: Mikey Ariel

Write the Docs Community Board Quarterly Update Q3-2023
=======================================================

> Greetings, documentarians! Mikey Ariel here, chairperson of the WTD community board, and I'm happy to share with you the third quarterly update for 2023. 

Reviewer: reword the "and " maybe, it reads odd to me.

> Greetings, documentarians! Mikey Ariel here, chairperson of the WTD community board, and I'm happy to share with you the third quarterly update for 2023. 

Author: ```suggestion
Greetings, documentarians! Mikey Ariel here, chairperson of the WTD community board. I'm happy to share with you the third quarterly update for 2023. 
```

To catch up on the previous quarterly updates, check out `Q1 blog post <https://www.writethedocs.org/blog/2023-Q1-community-board/>`_ and `Q2 blog post <https://www.writethedocs.org/blog/2023-Q2-community-board/>`_. 

Q3 quarterly meeting highlights
-------------------------------

Our third meeting covered several topics: 

> Meetup organizer mailing list and survey::

Author: ```suggestion
**Meetup organizer mailing list and survey**
```

>     Earlier this year, our Meetups team lead Rose Williams created a mailing list of all the meetup organizers, in an effort to formalize communication channels so that we can support the various meetups around the world more easily. Rose also sent out a survey to all the meetup organizers where they can indicate whether their meetup is still active, and submit ideas on how to evolve local and regional meetup structure. 

Reviewer 2: I think this is great, I think this might be a more clear explanation. Let me know if you agree!

"This year the Meetup team is focusing on updating organizer resources and better communication. To support our efforts the Meetup team lead, Rose Williams, created an organizer mailing list and distributed a meetup activity survey. The survey assessed meetup activity and collected ideas for evolving the local and regional meetup structure."

Author: I love it! Thanks for the suggestion.

>     Earlier this year, our Meetups team lead Rose Williams created a mailing list of all the meetup organizers, in an effort to formalize communication channels so that we can support the various meetups around the world more easily. Rose also sent out a survey to all the meetup organizers where they can indicate whether their meetup is still active, and submit ideas on how to evolve local and regional meetup structure. 

Author: ```suggestion
This year the Meetup team is focusing on updating organizer resources and streamlining communication. To support our efforts, the Meetup team lead Rose Williams created an organizer mailing list and distributed a meetup activity survey. The survey assessed meetup activity and collected ideas for evolving the local and regional meetup structure.
```

>     We appreciate all of the hard work that meetup organizers do to keep their communities engaged, and will continue to work with the meetup teams to adapt to the current the global landscape. If you want to start a new meetup or join an existing meetup, check out the `Meetups page <https://www.writethedocs.org/meetups/>`_ on our website.

Author: ```suggestion
    This year the Meetup team is focusing on updating organizer resources and streamlining communication. To support our efforts, the Meetup team lead Rose Williams created an organizer mailing list and distributed a meetup activity survey. The survey assessed meetup activity and collected ideas for evolving the local and regional meetup structure.
    
    We appreciate all of the hard work that meetup organizers do to keep their communities engaged, and will continue to work with the meetup teams to adapt to the current the global landscape. If you want to start a new meetup or join an existing meetup, check out the `Meetups page <https://www.writethedocs.org/meetups/>`_ on our website.
```

> Slack channel naming and structure::

Author: ```suggestion
**Slack channel naming and structure**
```

    Last year, the Slack Moderation team started working on a proposal to restructure the Slack channels and establish naming conventions. Since then, the initiative was set aside due to changing availability, but the community board agreed that there are community benefits to restarting that effort, so we will be crafting a WEP (Write the Docs Enhancement Proposal) and posting it soon on our `WEP repository <https://github.com/writethedocs/weps>`_ for community review. 

> Team lead communication channels::

Author: ```suggestion
**Team lead communication channels**
```

    As a part of our continuous improvements to the organizational administration, we decided to create official WTD Google accounts for all team leads. This will enable them to interact with the community with WTD email addresses, and provide them with the ability to create google groups for their teams. These groups can be used for mailing lists (such as the meetup organizer mailing list) and Google Drive access for team and event resources. 

> 2023 salary survey::

Author: ```suggestion
**2023 salary survey**
```

    Now that the `2022 salary survey <https://www.writethedocs.org/surveys/salary-survey/2022/>`_ is published, we're starting to work on the next survey with Kay Smoljak, our-long term survey expert! Expect the survey to be opened around the Atlantic conference timeframe. 

And as always, the board continues to review the community backlog to determine what are the most valuable efforts for the community in the short and long term. 

What is the community board
---------------------------

The `Community Board <https://www.writethedocs.org/team/#community-board>`_ exists to curate the overall strategic direction of the community teams.

The board members are team leads for each WTD team, as well as the conference chairs of our various conferences:

* Mikey Ariel (CoreOps, Brand and design, board chair)
* Eric Holscher (CoreOps, Portland chair)
* Samuel Wright (CoreOps, Infrastructure)

> * Sasha Romijn (CoreOps, Support, Atlantic Chair)

Reviewer 3: Just for consistency with the other chairs, not super important either way.
```suggestion
* Sasha Romijn (CoreOps, Support, Atlantic chair)
```
